app/views/my_module_comments/_comment.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="pull-right">
   <span class="text-muted"><%= l comment.created_at, format: '%H:%M' %></span>
-  <% if can_edit_module_comment(comment) || can_delete_module_comment(comment) %>
+  <% if can_manage_comment_in_module?(comment.becomes(Comment)) %>
```

Comment:
Yes, because in experiment level permissions we need to check three different types of comments (for module, result and step), so I've made just one permissions block for Comment type, instead of 3 block for each type separately, so in order to make it work, we need to cast the comment to its parent type, i.e. Comment.